hera_cal/apply_cal.py

```diff
@@ -327,9 +357,9 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
                         if np.all(data_flags[bl]):
                             redundant_weights[bl][:] = 0.
                 # redundantly average
-                utils.red_average(data=data, flags=data_flags, nsamples=data_nsamples,
-                                  reds=all_red_antpairs, wgts=redundant_weights, inplace=True,
-                                  propagate_flags=True)
+                data, data_flags, data_nsamples = utils.red_average(data=data, flags=data_flags, nsamples=data_nsamples,
```

Comment:
Why not use `inplace=True`? Seems like it would be more memory efficient, no?

Reply:
I now use inplace here.